projects/modals/src/public-api.ts

```diff
@@ -17,3 +17,9 @@ export * from './modules/modal/modal-instance';
 export * from './modules/modal/modal.interface';
 export * from './modules/modal/modal.module';
 export * from './modules/modal/modal.service';
+
+export { SkyConfirmComponent as λ1 } from './modules/confirm/confirm.component';
```

Comment:
Mind if we add the comment above these exports to be consistent with the other libraries?
Example: https://github.com/blackbaud/skyux-core/blob/5.0.0-next/projects/core/src/public-api.ts#L70-L71